packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -375,6 +376,7 @@ const getSimpleSeries = ({
                             formats[yFieldHash].format,
                             formats[yFieldHash].round,
                             value?.value?.[yFieldHash],
```

Comment:
out of scope but can we move value as the first argument?

Reply:
Added to the PR todo list after this one is merged.